contracts/hooks/examples/LimitOrder.sol

```diff
@@ -159,7 +159,7 @@ contract LimitOrder is BaseHook {
 
     function _fillEpoch(PoolKey calldata key, int24 lower, bool zeroForOne) internal {
         Epoch epoch = getEpoch(key, lower, zeroForOne);
-        if (!epoch.equals(EPOCH_DEFAULT)) {
+        if (!epoch.equals(EPOCH_NOT_INITIALIZED)) {
```

Comment:
hmm i actually find this harder to read..

if epoch does not equal epoch not initialized ? its like double negative

maybe could you provide some more context/motivation for this name change?

Reply:
i feel like it makes the logic easier to follow. it emphasizes that there is nothing in that specific epoch. "default" feels confusing. maybe change the constant to EPOCH_EMPTY?